recipes/drwav/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
+  "0.13.2":
```

Comment:
```suggestion
  "cci.2021.10.01":
```

Reply:
See my comment above. 0.13.2 is the latest release of dr_wav: https://github.com/mackron/dr_libs/blob/master/dr_wav.h#L3.

Reply:
Fair enough, I suggest adding an information then.